src/lib/yang/value.lua

```diff
@@ -79,7 +79,15 @@ function types.decimal64.tostring(val)
 end
 
 types.empty = unimplemented('empty')
-types.identityref = unimplemented('identityref')
+
+types.identityref = {}
+function types.identityref.parse(str, what)
+   return assert(str, 'missing value for '..what)
```

Comment:
We should add a comment that these references are expanded in the validation phase.